.github/workflows/base-wrapper.yml

```diff
@@ -33,10 +33,6 @@ on:
         type: string
         description: 'Mandrel packaging repository to be used'
         default: "graalvm/mandrel-packaging"
-      distribution:
-        type: string
-        description: 'Distribution to build, mandrel or graalvm'
-        default: "mandrel"
```

Comment:
Unfortunately we can't remove this since we need to be able to test both Graal and Mandrel builds